aiconfig-docs/docs/editor.md

```diff
@@ -0,0 +1,166 @@
+
+- [Set up the Editor](#set-up)
+- [Open the AIConfig Editor](#open)
+- [Edit and save AIConfigs](#edit-and-save)
+- [Run prompts](#run-prompts)
+- [Chain prompts](#chain-prompts)
+- [Create prompt templates](#prompt-templates)
+
+Want to get started quickly? Check out our [Getting Started Tutorial](./getting-started).
+
+---
+
+## Set Up {#set-up}
+
+1. Install the [AIConfig python package](https://pypi.org/project/python-aiconfig/) to use the AIConfig editor.
+
+<Tabs groupId="package-manager" queryString defaultValue={constants.defaultPythonPackageManager} values={constants.pythonPackageManagers}>
+<TabItem value="pip">
+
+```bash
+$ pip3 install python-aiconfig
+```
+
+</TabItem>
+
+<TabItem value="poetry">
+
+```bash
+$ poetry add python-aiconfig
+```
+
+</TabItem>
+
+</Tabs>
+
+:::note
+You need to install the python AIConfig package to create and edit your configs using the AIConfig Editor. You can still use the AIConfig Node SDK to interact with your config in your application code.
+:::
+
+2. Setup your API Keys required by the model providers.
+
+You will need to specify your API keys for the model providers (i.e. OpenAI, Google, HuggingFace) you plan to use. We recommend adding your API keys as environment variables so that they are accessible for all projects. The python library will automatically detect and use them without you having to write any code.
+
+<details> 
+    <summary>Example: Setup your OpenAI API Key as a environment variable (MacOS / Linux / Windows)</summary>
+    <div>
+        1. Get your OpenAI API Key: https://platform.openai.com/account/api-keys 
+        2. Open Terminal
+        3. Edit Bash Profile: Use the command `nano ~/.bash_profile` or `nano ~/.zshrc` (for newer MacOS versions) to open the profile file in a text editor.
+        4. Add Environment Variable: In the editor, add the line below, replacing *your-api-key-here* with your actual API key:
+        ```bash 
+        export OPENAI_API_KEY='your-api-key-here'
+        ```
+        5. Save and Exit: Press `Ctrl+O` followed by `ENTER` to write the change. Then `Ctrl+X` to close the editor.
+        6. Load Your Profile: Use the command `source ~/.bash_profile` or `source ~/.zshrc` to load the updated profile.
+        7. Verification: Verify the setup by typing `echo $OPENAI_API_KEY` in the terminal. It should display your API key.
+
+    </div>
+
+</details>
+
+## Open AIConfig Editor {#open}
+
+You can open the AIConfig Editor from your terminal to start prompting against models (saved as AIConfigs).
+
+1. Open Terminal
+2. Run this command: `aiconfig edit`
+
+This will open the AIConfig Editor in your default browser and in parallel create a new empty AIConfig JSON file in your current directory. Your work in the editor will be saved by default to `my_aiconfig.aiconfig.json`. Update the command to `aiconfig edit –{file_path_name}` if you want to save to a specified file path.
```

Comment:
cc @jonathanlastmileai , does this need to be `aiconfig edit --aiconfig-path={file_path_name` instead, or is `aiconfig edit –{file_path_name}` also good enough?